src/container/srv_target.c

```diff
@@ -84,10 +99,8 @@ cont_child_aggregate(struct ds_cont_child *cont, uint64_t *sleep)
 	int			tgt_id = dss_get_module_info()->dmi_tgt_id;
 	int			i, rc;
 
-	interval = (uint64_t)DAOS_AGG_THRESHOLD * NSEC_PER_SEC;
-	*sleep = interval;
-
-	if (dss_aggregation_disabled())
+	*sleep = NSEC_PER_SEC;
```

Comment:
Minor: In lazy mode, if there are active IOs, we'll schedule aggregation ULT once every second, I think we could enlarge this value a bit, like 5 seconds?

Reply:
OK